app/mailers/application_mailer.rb

```diff
@@ -1,4 +1,4 @@
 class ApplicationMailer < ActionMailer::Base
-  default from: "from@example.com"
-  layout "mailer"
+  default from: 'from@example.com'
+  layout 'mailer'
```

Comment:
- [Optional] Dear @Shedrack-Sunday, I noticed from your PR description, you ask to see the changes you made. Please understand you need to implement all the changes in the new/feature branch. This is because your changes are not reflected on the new/feature branch if they are already implemented on the old/dev branch. I hope this makes sense...

Reply:
Thanks for the note @TadesseDev